lifecycle_msgs/msg/Transition.msg

```diff
@@ -10,7 +10,9 @@ uint8 TRANSITION_CONFIGURE = 1
 uint8 TRANSITION_CLEANUP = 2
 uint8 TRANSITION_ACTIVATE = 3
 uint8 TRANSITION_DEACTIVATE = 4
-uint8 TRANSITION_SHUTDOWN = 5
+uint8 TRANSITION_UNCONFIGURED_SHUTDOWN  = 50
+uint8 TRANSITION_INACTIVE_SHUTDOWN = 51
+uint8 TRANSITION_ACTIVE_SHUTDOWN = 52
```

Comment:
Maybe `5` - `7` for the new transitions and bumping `TRANSITION_DESTROY` to `8`?